backend/locales/zh-TW.json

```diff
@@ -0,0 +1,183 @@
+    "error": "錯誤",
+    "plugin_error_uninstall": "如果您要解除安裝外掛程式，請在 Decky 選單，按下 {{-icon}}。",
+    "plugin_load_error": {
+      "message": "載入外掛程式 {{name}} 發生錯誤",
+      "toast": "{{name}} 載入出錯"
+    },
+    "plugin_uninstall": {
+      "button": "解除安裝",
+      "desc": "您確定要解除安裝 {{name}} 嗎？",
+      "title": "解除安裝 {{name}}"
+    },
+    "plugin_update_one": "可更新 1 個外掛程式！",
+    "plugin_update_other": "可更新 {{count}} 個外掛程式！"
+  },
+  "RemoteDebugging": {
+    "remote_cef": {
+      "desc": "允許您的網路中的任何人未經認證地存取 CEF 偵錯器",
+      "label": "允許 CEF 遠端偵錯"
+    }
+  },
+  "SettingsDeveloperIndex": {
+    "header_other": "其他",
+    "react_devtools": {
+      "desc": "啟用與執行 React DevTools 的電腦的連接。改變這個設定將重新載入 Steam。啟用前必須設定 IP 位址。",
+      "ip_label": "IP",
+      "label": "啟用 React DevTools"
+    },
+    "third_party_plugins": {
+      "button_install": "安裝",
+      "button_zip": "開啟",
+      "header": "第三方外掛程式",
+      "label_desc": "網址",
+      "label_url": "從網址安裝外掛程式",
+      "label_zip": "從 ZIP 檔案安裝外掛程式"
+    },
+    "toast_zip": {
+      "body": "安裝失敗！只支援 ZIP 檔案。",
+      "title": "Decky"
+    },
+    "valve_internal": {
+      "desc1": "啟用 Valve 內建開發人員選單。",
+      "desc2": "除非您知道它的作用，否則不要碰這個選單中的任何東西。",
+      "label": "啟用 Valve 內建"
+    }
+  },
+  "SettingsGeneralIndex": {
+    "about": {
+      "decky_version": "Decky 版本",
+      "header": "關於"
+    },
+    "beta": {
+      "header": "參與測試"
+    },
+    "developer_mode": {
+      "desc": "啟用 Decky 的開發人員模式。",
+      "label": "開發人員模式"
+    },
+    "other": {
+      "header": "其他"
+    },
+    "updates": {
+      "header": "更新"
+    }
+  },
+  "SettingsIndex": {
+    "developer_title": "開發人員",
+    "general_title": "一般",
+    "navbar_settings": "Decky 設定",
+    "plugins_title": "外掛程式"
+  },
+  "Store": {
+    "store_contrib": {
+      "desc": "如果您想為 Decky 外掛程式商店做貢獻，請查看 GitHub 上的 SteamDeckHomebrew/decky-plugin-template 儲存庫。README 中提供了有關開發和發佈的資訊。",
+      "label": "貢獻"
+    },
+    "store_filter": {
+      "label": "過濾",
+      "label_def": "全部"
+    },
+    "store_search": {
+      "label": "搜尋"
+    },
+    "store_sort": {
+      "label": "排序",
+      "label_def": "最後更新 (最新)"
+    },
+    "store_source": {
+      "desc": "所有外掛程式原始碼可以在 GitHub 的 SteamDeckHomebrew/decky-plugin-database repository 查看。",
+      "label": "原始碼"
+    },
+    "store_tabs": {
+      "about": "關於",
+      "alph_asce": "依字母排序 (Z 到 A)",
+      "alph_desc": "依字母排序 (A 到 Z)",
+      "title": "瀏覽"
+    },
+    "store_testing_cta": "請考慮測試新的外掛程式來幫助 Decky Loader 團隊！"
+  },
+  "StoreSelect": {
+    "custom_store": {
+      "label": "自訂商店",
+      "url_label": "網址"
+    },
+    "store_channel": {
+      "custom": "自訂",
+      "default": "預設",
+      "label": "商店頻道",
+      "testing": "測試"
+    }
+  },
+  "Updater": {
+    "decky_updates": "Decky 更新",
+    "no_patch_notes_desc": "這個版本沒有更新日誌",
+    "patch_notes_desc": "更新日誌",
+    "updates": {
+      "check_button": "檢查更新",
+      "checking": "'正在檢查",
```

Comment:
The ' after the " was a typo in my english translation, after that is removed it looks good to merge to me.